apps/src/code-studio/pd/workshop_dashboard/components/survey_results/survey_rollup_table.jsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import {Table} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+const questionCategories = [
+  'facilitator_effectiveness',
+  'teacher_engagement',
+  'overall_success'
+];
+
+export class SurveyRollupTable extends React.Component {
+  static propTypes = {
+    rollups: PropTypes.object.isRequired,
+    questions: PropTypes.object.isRequired,
+    facilitators: PropTypes.object.isRequired
```

Comment:
I don't know about `PropTypes.shape`, it'll be handy later!
Using it here is tricky for `rollups` object because its key names are not constants. The keys are dynamic and designed to distinguish different roll-up scenarios (e.g facilitator_X_this_workshop, facilitator_Y_all_workshops).

`questions` and `facilitators` are simple object with property values are strings.